dotnet/samples/Demos/VectorStoreRAG/README.md

```diff
@@ -55,6 +59,7 @@ For Azure OpenAI Embeddings, you need to add the following secrets:
 
 ```cli
 dotnet user-secrets set "AIServices:AzureOpenAIEmbeddings:Endpoint" "https://<yourservice>.openai.azure.com"
+dotnet user-secrets set "AIServices:AzureOpenAIEmbeddings:DeploymentName" "text-embedding-ada-002"
```

Comment:
As above

Reply:
Sure, I was in between wanting to give people an example and it being confusing having an actual value in there.  Happy to add the placeholder.